libs/sdk-ui-filters/src/AttributeFilter/hooks/useAttributeFilterController.ts

```diff
@@ -433,17 +459,44 @@ function useCallbacks(
     const onOpen = useCallback(() => {
         if (shouldReloadElements) {
             handler.loadInitialElementsPage(RESET_CORRELATION);
+            handler.loadIrrelevantElements(IRRELEVANT_SELECTION);
             setShouldReloadElements(false);
         }
     }, [handler, shouldReloadElements, setShouldReloadElements]);
 
+    const onShowFilteredElements = useCallback(() => {
+        if (supportsShowingFilteredElements) {
+            handler.setIncludeLimitingFilters(false);
```

Comment:
Could this be replaced with `handler.setLimitingAttributeFilters([])` + local state (show / hide)? It seems as a similar case as before - dumb setter / getter that is not affecting the handler logic inside it at all :)